source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2243,6 +2248,10 @@ export abstract class Action {
     parse(json: any) {
         this.id = json["id"];
         this.title = json["title"];
+        this.isPrimaryAction = false;
+        if (json["isPrimaryAction"] != null) {
```

Comment:
JSON property name should also be isPrimary.
We shouldn't parse isPrimary in the library, as it is not an official Adaptive property. It's ok to have support for it, but it is up to the host to custom parse it.

Reply:
This is outdated code. It is not from latest commit